src/tests/ftest/daos_test/dfuse.yaml

```
@@ -0,0 +1,38 @@
+hosts:
+  test_servers:
+    - server-A
+  test_clients:
+    - client-B
+timeout: 60
+pool:
+  scm_size: 1G
+  control_method: dmg
+server_config:
+  name: daos_server
+container:
+  type: POSIX
+  control_method: daos
+dfuse: !mux
+  writeback:
+    # File metadata is incorrect after write with writeback caching and IL.
+    !filter-only : "/run/intercept/off"  # yamllint disable-line rule:colons
+    mount_dir: "/tmp/daos_dfuse/"
+    name: "writeback"
+  writethrough:
+    mount_dir: "/tmp/daos_dfuse/"
+    disable_wb_cache: true
+    name: "writethrough"
+  metadata:
+    mount_dir: "/tmp/daos_dfuse/"
+    disable_wb_cache: true
+    name: "metadata"
+  nocache:
+    mount_dir: "/tmp/daos_dfuse/"
+    disable_wb_cache: true
+    disable_caching: true
+    name: "nocache"
```

Optional - Just FYI that you can do something like this for shared config values
```suggestion
dfuse:
  mount_dir: "/tmp/daos_dfuse/"
  io_mode: !mux
    writeback:
      # File metadata is incorrect after write with writeback caching and IL.
      !filter-only : "/run/intercept/off"  # yamllint disable-line rule:colons
      name: "writeback"
    writethrough:
      disable_wb_cache: true
      name: "writethrough"
    metadata:
      disable_wb_cache: true
      name: "metadata"
    nocache:
      disable_wb_cache: true
      disable_caching: true
      name: "nocache"
```

---

I might experiment with this later once I've got my head around it but I'm blocked on these tests landing and CI turnaround is slow so I won't make changes now.